bibtex-actions.el

```diff
@@ -522,14 +522,16 @@ With prefix, rebuild the cache before offering candidates."
 (defun bibtex-actions-run-default-action (keys)
   "Run the default action `bibtex-actions-default-action' on KEYS."
   (funcall bibtex-actions-default-action
-           (if (stringp keys) (split-string keys " & ") keys)))
+           (if (stringp keys)
+               (split-string keys " & ")
+             (split-string (cdr keys) " & "))))
```

Comment:
Why do we need both dwim and run-default? Couldn't we just add an optional keys arg on the former?